_docs/individual-payments.md

```diff
@@ -40,19 +40,19 @@ The valid `bank_code` values are:
 {% capture data-raw %}
 ```
 Access Bank: 044
+Citi Bank: 023
```

Comment:
If this is a new bank we need to support we have to add dev work to suport it with our current payment providers